Dockerfile

```diff
@@ -71,6 +71,8 @@ RUN install-pip pipenv 2020.11.15
 
 # renovate: datasource=github-releases lookupName=python-poetry/poetry
 RUN install-tool poetry 1.1.11
+# Workaround until https://github.com/python-poetry/poetry/issues/4688 is fixed
+/usr/local/poetry/*/venv/bin/pip install cachecontrol==0.12.6
```

Comment:
```suggestion
RUN /usr/local/poetry/1.1.11/venv/bin/pip install cachecontrol==0.12.6
```

Reply:
Oops, fixed.

Reply:
I think this should only be done for 1.1.11 and intentionally fail on next upgrade, where poetry is hopefully fixed.